fem/tmop_tools.cpp

```diff
@@ -381,7 +430,7 @@ double TMOPNewtonSolver::ComputeScalingFactor(const Vector &x,
       if (energy_out > 1.2*energy_in || std::isnan(energy_out) != 0)
       {
          if (print_level >= 0)
-         { mfem::out << "Scale = " << scale << " Increasing energy.\n"; }
+         { mfem::out << "Scale = " << scale << " " << energy_out << " " << "Increasing energy.\n"; }
```

Comment:
Were you debugging something or actually changing the output? Indicate that the second number is the energy / the line is too long.

Reply:
I think this is from debugging. I will undo this change.